### PR TITLE
Run well

### PR DESCRIPTION
**Pull request by mgcam**
Built on the top of https://github.com/wtsi-npg/npg_langqc/pull/55

### CODE REVIEW

File: tests/test_filtered_wells.py [hunk 1]
```diff
@@ -48,22 +51,22 @@ def assert_filtered_inbox_equals_expected(response, expected_data):
...
-        for well_info in result.wells:
-            wells[well_info.label] = (
-                well_info.qc_status.qc_state
-                if well_info.qc_status is not None
-                else None
-            )
-        actual_data[result.run_name] = wells
-
+    results = response.json()
+    assert type(results) is list
```
Comment (nerdstrike) [inline tests/test_filtered_wells.py:55]: Is there any need to check data types on a pydantic-constrained return value?

File: tests/test_filtered_wells.py [hunk 2]
```diff
@@ -48,22 +51,22 @@ def assert_filtered_inbox_equals_expected(response, expected_data):
...
+    for result in results:
+        assert type(result) is dict
+        rwell = result["run_name"] + ":" + result["well"]["label"]
+        qc_state = (
+            result["well"]["qc_status"]["qc_state"]
+            if result["well"]["qc_status"] is not None
+            else None
+        )
+        actual_data.append({rwell: qc_state})
+    print(actual_data)
```
Comment (nerdstrike) [inline tests/test_filtered_wells.py:68]: Feel free to expunge these prints.

**Review by nerdstrike — CHANGES_REQUESTED** [review]

File: tests/test_filtered_wells.py (hunk 1 above)
Comment (mgcam) [reply tests/test_filtered_wells.py:55]: The list is explicitly declared as the return value of the end point. We do not have a model for this list.

File: tests/test_filtered_wells.py (hunk 2 above)
Comment (mgcam) [reply tests/test_filtered_wells.py:68]: Thanks, forgot

**Review by nerdstrike — APPROVED** [review]

**Pull request merged**